### PR TITLE
Add initial Compose experiment

### PR DESCRIPTION
**Pull request by pavlospt**
_Per title_

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -21,6 +22,7 @@ buildscript {
     dependencies {
         classpath(GradlePlugins.ANDROID)
         classpath(kotlin(module = GradlePlugins.Kotlin.ID, version = GradlePlugins.Kotlin.VERSION))
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
```
Comment (charbgr) [inline build.gradle.kts]: :mag: I think this is not correct? @pavlospt

Comment (pavlospt) [reply build.gradle.kts]: 🤔🤔🤔🤔

Comment (pavlospt) [reply build.gradle.kts]: @charbgr fixed!

Comment (charbgr) [reply build.gradle.kts]: 86225012-5130e680-bb92-11ea-8631-d8622645d97f

**Pull request closed without merging**